.github/workflows/release.yml

```diff
@@ -15,19 +15,49 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.19
+          go-version: 1.23.1
 
       - name: Import GPG key
         id: import_gpg
         uses: crazy-max/ghaction-import-gpg@v5.1.0
         with:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
 
+      - name: Install spacectl
+        uses: spacelift-io/setup-spacectl@main
```

Comment:
is `main` a stable reference point?

Typically when referencing github actions, you'd reference a release tag, or if you're developing, a feature branch.

Targetting the main branch doesn't guarantee stability, so I'd recommend referencing a release tag

Reply:
same here but I've updated